src/components/react_multi_select.scss

```diff
@@ -0,0 +1,116 @@
+@import "../resources/globals";
+
+.wrapper {
```

Comment:
Can we flatten this or is it a limitation of one of our dependencies?

Reply:
We can flatten it. Why not use it as a hierarchy?

Reply:
I don't mind it but is is meaningless when using css modules with hash, we are importing a specific classes and the hierarchy adds nothing.

Reply:
done